src/api.py

```diff
@@ -196,19 +198,39 @@ def recitation(self):
                 else:
                     raise EditorNotFound()
             except EditorNotFound:
-                print("Text editor not found; set the 'EDITOR'" +
+                if platform.system() == "Windows":
+                    editor = "notepad"
+                    windowsfilename = f"{self.target.reference}"
+                    windowsfilename = windowsfilename.replace(":","-")
+                else:
+                    print("Text editor not found; set the 'EDITOR'" +
                       "environmental variable and try again")
-                exit()
+                    exit()
+
+        if platform.system() == "Windows":
+            self.filename = self.cache_path / windowsfilename
+        else:
+            self.filename = self.cache_path / f"{self.target.reference}"
 
-        self.filename = self.cache_path / f"{self.target.reference}"
-        subprocess.run([editor, self.filename])
+        if os.path.isfile(self.filename) == False:
+            with open(self.filename, "w") as file:
+                subprocess.run([editor, self.filename])
+        else:
+            subprocess.run([editor, self.filename])
+
+        withextension = f"{self.filename}" + ".txt"
```

Comment:
[NIT] I've been using snake case for variable names up to this point. Perhaps:
```suggestion
        with_extension = f"{self.filename}" + ".txt"
```